flexmeasures/data/tests/test_scheduling_repeated_jobs.py

```diff
@@ -0,0 +1,215 @@
+# flake8: noqa: E402
```

Comment:
Just curious: what's the reason for masking this module from flake8?

Reply:
I copied it from `test_scheduling_jobs.py`.

From [SO](https://stackoverflow.com/questions/48498139/how-to-fix-issues-with-e402):

> E402 refers to imports only being at the top of a file. This is to stop the following:

Probably, we don't require this exception `test_scheduling_jobs.py` because there are no imports but in the top of the file.

Reply:
Feel free to remove it if our automation pipeline doesn't complain.